### ISSUE TITLE
Redering errors on cips.cardano.org

### ISSUE THREAD

**Issue by kwxm** [cardano-foundation/cf-cip-frontend#28]
[CIP-0109 ](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0109#specification) contains the following markdown:

```
Here we have that the base $b$ is an integer, the exponent $e$ a non-negative integer and the modulus $m$ a positive integer. To be more specific, we want $e$ to be larger or equal to zero, and $m$ is larger than zero. This is because what we are effectively doing, is multiplying $e$ copies of $b$ and a modulo reduction by $m$. In this context, multiplying a negative number of copies of $b$ has no definition.
```

The text within `$...$` is rendered as expected on GitHub:

![CIP-0109-GitHub](https://github.com/cardano-foundation/cf-cip-frontend/assets/2124565/5b2bdc24-5c00-4e1a-88d1-a2546b3f607e)

However on `cips.cardano.org` the symbols inside `$...$` appear twice:

![CIP-0109-web](https://github.com/cardano-foundation/cf-cip-frontend/assets/2124565/b4686b13-def8-4444-9e6b-303b0e1d95e1)

Something similar happens in CIP-1694:

![CIP-1694-web](https://github.com/cardano-foundation/cf-cip-frontend/assets/2124565/9b4fdb2e-0dd1-4c70-8d68-13fbbb18792f)

This is minor, but I thought it was worth pointing out.  Maybe people aren't supposed to be using TeX in CIPs.

**Comment by fill-the-fill** [comment]
resolved in #31 , should be live soon  :)

**Comment by fill-the-fill** [comment]
resolved in https://github.com/cardano-foundation/cf-cip-frontend/pull/31 , should be live soon :)
